client/landing/jetpack-cloud/components/scan-threats/index.tsx

```diff
@@ -159,6 +159,14 @@ const ScanThreats = ( { error, site, threats }: Props ) => {
 		[ updatingThreats ]
 	);
 
+	const onOpenTrackEvent = React.useCallback( () => {
```

Comment:
I believe we can even take advantage of `useTrackCallback` here.

Reply:
Good catch! Thanks @monsieur-z .